Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,59 @@
+//Solved by Arion Hamel
+#include "Person.h"
+#include <iostream>
+#include <string>
+#include <cstdlib>
+
+// -- Constructors --
+Person::Person() {
+    name_ = "";
+    age_ = new int(0);
+}
+
+Person::Person(const std::string& name, int age) {
+    name_ = name;
+    age_ = new int(age);
+}
+
+// -- Descontructor --
+Person::~Person() {
+    name_.clear();

```

Comment:
Unnecessary because we do not need to manually deallocate the string. If we do `delete <insert person object here>`, name_ is taken care of. We only have to worry about age_.